api/catalog/api/utils/pagination.py

```diff
@@ -1,48 +1,18 @@
 from rest_framework.pagination import PageNumberPagination
 from rest_framework.response import Response
 
-from catalog.api.utils.exceptions import get_api_exception
-
 
 class StandardPagination(PageNumberPagination):
-    page_size_query_param = "page_size"
-    page_query_param = "page"
+    page_size_query_param = None
+    page_query_param = None
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.result_count = None  # populated later
         self.page_count = None  # populated later
 
-        self._page_size = 20
-        self._page = None
-
-    @property
-    def page_size(self):
-        """the number of results to show in one page"""
-        return self._page_size
-
-    @page_size.setter
-    def page_size(self, value):
-        if value is None or not str(value).isnumeric():
-            return
-        value = int(value)  # convert str params to int
-        if value <= 0 or value > 500:
-            raise get_api_exception("Page size must be between 0 & 500.", 400)
-        self._page_size = value
-
-    @property
-    def page(self):
-        """the current page number being served"""
-        return self._page
-
-    @page.setter
-    def page(self, value):
-        if value is None or not str(value).isnumeric():
-            value = 1
-        value = int(value)  # convert str params to int
-        if value <= 0:
-            raise get_api_exception("Page must be greater than 0.", 400)
-        self._page = value
+        self.page_size = 20
```

Comment:
Why are we removing the page size and page setting?
Wouldn't this be a breaking change for the API?

Reply:
Yeah, by adding `page` and `page_size` to the serializer, we're able to 
- validate their limits using `min_value` and `max_value` in the serializer
- have that information passed directly to the search controller without manual effort